compass/landice/tests/ismip6_forcing/ocean_basal/process_basal_melt.py

```diff
@@ -0,0 +1,230 @@
+                            target=os.path.join(base_path_ismip6,
+                                                self._file_basin))
+
+        input_file_list = self._files_coeff
+        for file in input_file_list:
+            self.add_input_file(filename=os.path.basename(file),
+                                target=os.path.join(base_path_ismip6, file))
+            self.add_output_file(filename=f"processed_basin_and_"
+                                          f"{os.path.basename(file)}")
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        # logger = self.logger
+        config = self.config
+        section = config['ismip6_ais']
+        mali_mesh_name = section.get('mali_mesh_name')
+        mali_mesh_file = section.get('mali_mesh_file')
+        output_base_path = section.get('output_base_path')
+
+        section = config['ismip6_ais_ocean_basal']
+        method_remap = section.get('method_remap')
+
+        # combine, interpolate and rename the basin file and deltaT0_gamma0
+        # ismip6 input files
+        combined_file_temp = "combined.nc"  # temporary file names
+        remapped_file_temp = "remapped.nc"
+
+        # call the function that combines data
+        # logger.info = ('calling combine_ismip6_inputfiles')
+        input_file_list = self._files_coeff
+        i = 0
+        for file in input_file_list:
+            print(f"processing the input file {os.path.basename(file)}")
+            i += 1
+            self.combine_ismip6_inputfiles(os.path.basename(self._file_basin),
+                                           os.path.basename(file),
+                                           combined_file_temp)
+
+            # remap the input forcing file.
+            print("Calling the remapping function...")
+            self.remap_ismip6BasalMelt_to_mali(combined_file_temp,
+                                               remapped_file_temp,
+                                               mali_mesh_name,
+                                               mali_mesh_file, method_remap)
+
+            output_file = f"processed_basin_and_{os.path.basename(file)}"
+
+            # rename the ismip6 variables to MALI variables
+            print("Renaming the ismip6 variables to mali variable names...")
+            self.rename_ismip6BasalMelt_to_mali_vars(remapped_file_temp,
+                                                     output_file)
+
+            print("Remapping and renamping process done successfully. "
+                  "Removing the temporary files 'combined.nc' "
+                  "and 'remapped.nc'")
+
+            # remove the temporary combined file
+            os.remove(combined_file_temp)
+            os.remove(remapped_file_temp)
+
+            # place the output file in appropriate directory
+            if output_base_path == "NotAvailable":
+                return
+
+            output_path = f'{output_base_path}/basal_melt/parametrizations/'
```

Comment:
Either `parameterizations` or `parametrizations` would be correct but the directory on GHub is `parameterizations` (with an extra `e`).

```suggestion
            output_path = f'{output_base_path}/basal_melt/parameterizations/'
```